packages/bp/src/index.ts

```diff
@@ -97,7 +97,7 @@ try {
     process.env.CLOUD_OAUTH_ENDPOINT ||
     (localCloud ? 'http://localhost:4444/oauth2/token' : 'https://oauth.botpress.dev/oauth2/token')
   process.CLOUD_NLU_ENDPOINT =
-    process.env.CLOUD_NLU_ENDPOINT || (localCloud ? 'http://localhost:3200' : 'https://nlu.botpress.dev')
+    process.env.CLOUD_NLU_ENDPOINT || (localCloud ? 'http://localhost:3200' : 'https://nlu-builder.botpress.dev')
```

Comment:
what is local cloud ?

Reply:
It's the CLOUD_LOCAL env variable. You can set it to true to default to localhost ports instead